### ISSUE TITLE
Make Frontmatter Vars available in Layout

### ISSUE THREAD

**Issue by beberlei** [Pawka/phrozn#13]
I want to put the title of a post into the title, but the variables are not defined in the layout.

**Comment by dancrew32** [comment]
I would also like to see this! Having the page title show up in layout is my main concern.

**Comment by youngj** [comment]
I needed this too so I implemented it. A one-line change to the return statement in applyLayout in Site/View/Base.php is sufficient:

```
    return $layout->render(array(
        'content' => $content,
        'entry' => $vars['page'],
    ));
```

Then in the layout you can access any of the page variables (e.g. title) like this:

{{ entry.title }}

**Comment by ajgarlag** [comment]
The patch proposed by youngj was merged into dev branch https://github.com/farazdagi/phrozn/pull/21.

This issue can be closed.

**Comment by farazdagi** [comment]
Available as of version 0.3.3 
Thanks for the patch!
